echopype/qc/api.py

```diff
@@ -55,7 +55,8 @@ def coerce_increasing_time(
     would remain undisturbed.
     """
 
-    ds[time_name] = _clean_ping_time(ds[time_name].values, local_win_len=local_win_len)
+    ping_time_new = _clean_ping_time(ds[time_name].values, local_win_len=local_win_len)
+    ds[time_name].values[:] = ping_time_new
```

Comment:
Why does the operation need to be broken into 2 pieces?

Reply:
Are you suggesting that we do the following? If so, I do not have a problem with that. 

`ds[time_name].values[:] = _clean_ping_time(ds[time_name].values, local_win_len=local_win_len)`

Reply:
I was just wondering why the previous one before the change doesn't work. And the oneliner you put looks good.

Reply:
Ah, I see. The function `_clean_ping_time` returns a numpy array and in the previous version we assign this to the `ds` variable `time_name`. For this reason, we lose all attributes/encodings (if they exist). What we really want to do is just replace the values. 

> And the oneliner you put looks good.

Ok, I will change that into a oneliner.

Reply:
Thanks for the explanation! I completely missed the attributes side of things.